docs/03 - Implementing the Anchor Server/Communication/schemas.yml

```diff
@@ -0,0 +1,489 @@
+        id:
+          type: string
+          description: The ID of the transaction in the Stellar network.
+        memo:
+          type: string
+          description: The memo of the transaction in the Stellar network.
+        memo_type:
+          type: string
+          description: The memo type of the transaction in the Stellar network. Should be present if memo is not null.
+          enum:
+            - text
+            - hash
+            - id
+        created_at:
+          type: string
+          format: date-time
+          description: The time the transaction was registered in the Stellar network.
+        envelope:
+          type: string
+          description: The transaction envelope, containing all the transaction information.
+        payments:
+          type: array
+          items:
+            type: object
+            required:
+              - id
+              - payment_type
+              - source_account
+              - destination_account
+              - amount
+            properties:
+              id:
+                type: string
+                description: The ID of the payment in the Stellar Network.
+              payment_type:
+                type: string
+                description: The type of payment in the Stellar Network.
+                enum:
+                  - payment
+                  - path_payment
+                default: payment
+              source_account:
+                type: string
+                description: The account being debited in the Stellar Network.
+              destination_account:
+                type: string
+                description: The account being credited in the Stellar Network.
+              amount:
+                $ref: '#/components/schemas/Amount'
+    Error:
+      type: object
+      properties:
+        error:
+          type: string
+        id:
+          type: string
+      required:
+        - error
+
+    Event:
+      type: object
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+          enum:
+            - transaction_created
+            - transaction_status_changed
+            - transaction_error
+            - quote_created - a quote was created via the SEP38 API
+          description: |
+            The transaction event type. Can be one of the following:
+            - `transaction_created` - a transaction was created through the SEP endpoints. The payload is in the `transaction` field.
+            - `transaction_status_changed` - the status of a transaction has changed. The payload is in the `transaction` field.
+            - `transaction_error` - error processing the transaction. The payload is in the `transaction` field.
+            - `quote_created - a quote was created via the SEP38 API. The payload is in the `quote` field.
+        transaction:
+          $ref: '#/components/schemas/Transaction'
+        quote:
+          $ref: '#/components/schemas/Quote'
```

Comment:
Just double checking that you don't want to add a generic container key for the event payload? Something like `data` or `payload`?

Reply:
I think the generic container key `data` is not necessary. Do you have any good use case for the `data` field?

Reply:
I've just seen it used in other event systems, like Circle's. Its not strictly necessary but it does provide a separation between the event scheme (`id`, `type`, & `data`) vs. the schema specific to the event type, which is inside `data`. If we ever had an event with lots of top-level attributes, then it would be nicer to have a container attribute.

I don't feel strongly though, so I'll leave it to you.

Reply:
Thanks Jake. Appreciated your insight very much. I guess we may always nest extra top-level attributes. My gut feeling is to keep it simple at this moment. 

We still have some time until we release the AP and we may always revisit this issue.